src/SFA.DAS.ApprenticeAccounts.Web/Pages/NotificationSettings.cshtml

```diff
@@ -0,0 +1,81 @@
+﻿@page
+@model NotificationSettingsModel
+@{
+    ViewData["Title"] = "Change your email notifications settings";
+}
+<form method="post">
+    <a href="@Model.BackLink" class="govuk-back-link">Return to home</a>
+
+    @if (Model.SubmittedPreferences)
+    {
+        <partial name="_EmailNotificationSettingsSaved", model="@Model"/>
+    }
+
+    <partial name="ValidationSummary"/>
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
+                    <legend class="govuk-fieldset__legend">
+                        <h1 class="govuk-heading-l">
+                            @ViewData["Title"]
+                        </h1>
+                        <p>
```

Comment:
Can we move these 2 paragraphs out of the `<legend>` - just below the closing `</legend>`